packages/bpk-component-ticket/src/BpkTicket.js

```diff
@@ -16,132 +16,144 @@
  * limitations under the License.
  */
 
+/* @flow */
+
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { type Node } from 'react';
 import { cssModules } from 'bpk-react-utils';
 
 import STYLES from './bpk-ticket.scss';
 
 const getClassName = cssModules(STYLES);
 
-const isIE9 = () => {
```

Comment:
💥

Reply:
I can't tell if this is a positive or negative comment 😂